src/plugins/analysis/elf_analysis/code/elf_analysis.py

```diff
@@ -33,12 +33,14 @@ class AnalysisPlugin(AnalysisBasePlugin):
     def process_object(self, file_object):
         try:
             elf_dict, parsed_binary = self._analyze_elf(file_object)
-            file_object.processed_analysis[self.NAME] = {'Output': elf_dict}
             self.create_tags(parsed_binary, file_object)
-            file_object.processed_analysis[self.NAME]['summary'] = list(elf_dict.keys())
+            file_object.processed_analysis[self.NAME] = {
+                'result': {'Output': elf_dict},
```

Comment:
Yes but it would break compatibility with old results :frowning_face: 
We could change it in the template, too, but then we need to handle both cases which makes the template code uglier. Originally, I wanted to simply replace the `'Output'` key with `'results'`.